app/decorators/include_role_space_decorator.rb

```diff
@@ -7,11 +7,13 @@ def match?(include_params)
 
       def decorate(hash, roles)
         hash[:included] ||= {}
-        space_guids = roles.map(&:space_guid).uniq
-        spaces = Space.where(guid: space_guids).order(:created_at).
-                 eager(Presenters::V3::SpacePresenter.associated_resources).all
+        space_ids = roles.map { |role| role.space_id if role.for_space? }.compact.uniq
```

Comment:
might be a bit cleaner to do something like

```
roles.select(&:for_space?).map(&:space_id).uniq
```

Reply:
Yes, that's better. Thanks!